app/models.py

```diff
@@ -821,13 +821,13 @@ class ServiceInboundApi(BaseModel, Versioned):
     @property
     def bearer_token(self):
         if self._bearer_token:
-            return signer.verify(self._bearer_token)
+            return signer.verify_bearer_token(self._bearer_token)
```

Comment:
Regarding to my [previous comment](https://github.com/cds-snc/notification-api/pull/1782/files#r1136994473), this would rather be:
```
return signer_bearer_token.verify(self._bearer_token)
```